### ISSUE TITLE
Always show "0" relative images

### ISSUE THREAD

**Issue by hullymoly** [WangDaYeeeeee/Mysplash#33]
When I click more button always 0 relative photos. Please fix them.

**Comment by WangDaYeeeeee** [comment]
Yeah, Unsplash doesn't provide the data about relative photos, I will alter the user interface.
